frontend/package.json

```diff
@@ -171,7 +171,7 @@
     "graphql": "^14.0.0",
     "history": "^4.9.0",
     "hoist-non-react-statics": "3.x",
-    "i18next": "^21.8.14",
+    "i18next": "21.10.0",
```

Comment:
I think the changes in this file are the only ones needed to upgrade to the minor version.
I'm not sure why the other files are impacted. Could it be because you used the branch for a higher version upgrade with previous changes? I made only these changes in this[ commit](https://github.com/openshift/console/compare/main...cajieh:i18next-update-1?expand=1), and I have a clean `yarn build `and `yarn lint`. I suggest creating a new test branch, making these changes, and then running `yarn`, `yarn build`, and `yarn lint` confirm.